packages/gatsby-transformer-markdown-references/index.d.ts

```diff
@@ -0,0 +1 @@
+export * from "./lib/markdown-utils";
```

Comment:
I don't think this will have any typing information if there aren't the sources

Reply:
> I don't think this will have any typing information if there aren't the sources

Sorry, I didn't understand this sentence.

`/index.js` contains

```js
module.exports = require("./lib/markdown-utils");
```

so it exports everything from _./lib/markdown-utils.js_. 

I'm only adding a missing file to let TypeScript know about it and export all types from _**./lib/markdown-utils.d.ts**_

![image](https://user-images.githubusercontent.com/15332326/93124742-61a12980-f6ca-11ea-8a42-7d85e305b90d.png)
![image](https://user-images.githubusercontent.com/15332326/93124776-72519f80-f6ca-11ea-8468-6bdc09d43b06.png)

---

Temporary workaround for this is adding following code to user's `src/missing.d.ts` file.

```ts
declare module "gatsby-transformer-markdown-references" {
  export * from "gatsby-transformer-markdown-references/lib/markdown-utils";
}
```

Reply:
Oh right I forgot I was already generating the types, never mind sorry